sql/hive/pom.xml

```diff
@@ -133,7 +133,6 @@
                 </goals>
                 <configuration>
                   <sources>
-                    <source>src/test/scala</source>
```

Comment:
OK as long as we're sure this adds to rather than overrides `<sources>` from the parent.